tests/container_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env pytest -vs
+"""Tests for example container."""
```

Comment:
Replace template text from skeleton:
```suggestion
"""Tests for Domain Manager API container."""
```

Reply:
Fixed in this commit https://github.com/cisagov/domain-manager-api/pull/194/commits/bdf4957541d689ea3583c506bc5ecf22ffe5790f